# Cause "mrfast --help" or "mrfast --version" to have a non-error (0) exit code
--- CommandLineParser.c
+++ CommandLineParser.c
@@ -205,10 +205,10 @@
 	case 'h':
 	  printHelp();
-	  return 0;
+	  exit(0);
 	  break;
 	case 'v':
 	  fprintf(stderr, "mrFAST %s.%s with FastHASH\n", versionNumber, versionNumberF);
-	  return 0;
+	  exit(0);
 	  break;				
 	case '?': 
 	  fprintf(stderr, "Unknown parameter: %s\n", longOptions[index].name);
